### DISCUSSION TITLE
September 21st 2023 PowerShell Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#20148] [Community Call]
Agenda (more to come)
- PS Adapters Release @StevenBucher98 
- WinGet Feedback Provider @carlos-zamora  @crutkas
- Experimental Features for 7.4 @SteveL-MSFT 
- PSresourceGet rc release @anamnavi @SydneyhSmith @alerickson 
- Docs update @sdwheeler 
- VS Code pre-release(s) (and upcoming stable release) @SydneyhSmith 
- DSC alpha.2 @mgreenegit @SteveL-MSFT 
- PSReadLine GA - @StevenBucher98 
- Gallery potential intermittent connection issues on Thursday (moving prod to Front Door) and stats update @SydneyhSmith 
- Ignite @SydneyhSmith @StevenBucher98 
- November End-of-Year Community Call (2nd Thursday of the month)  @SteveL-MSFT 
- December Community Call  @SteveL-MSFT

**Comment by mgreenegit** [comment]
New PowerToy that adds the ability to detect failed commands in PowerShell and suggest a package to install using WinGet. This PowerToy is powered by a PowerShell module that leverages the new feedback subsystem.

Quick demo and feedback
https://github.com/microsoft/PowerToys/pull/26319

**Comment by anamnavi** [comment]
Other agenda topics:

- PSresourceGet rc release
- Docs update
- VS Code pre-release(s) (and upcoming stable release)
- DSC alpha.2
- Gallery potential intermittent  connection issues on Thursday (moving prod to Front Door) and stats update

**Comment by theJasonHelmick** [comment]
Can we please add
* November End-of-Year Community Call (2nd Thursday of the month) - Steve Lee
* December Community Call - Steve Lee

**Comment by StartAutomating** [comment]
If there's time, I can talk about:

* [Pacific PowerShell User Group](https://www.meetup.com/pacific-powershell-user-group/)
* More on [Posh](https://github.com/StartAutomating/Posh)
* [PSA](https://github.com/StartAutomating/PSA) - A new module for Announcements and BlueSky
* This month [HelpOut](https://github.com/StartAutomating/HelpOut) got a nice batch of changes that let us document ETS more effectively.
* There's also the useful module I use everyday, [ugit](https://github.com/StartAutomating/ugit)

**Comment by AspenForester** [comment]
Just as a heads-up: I added info to https://github.com/Azure/azure-powershell/issues/14416 this week.  It seems to me that there's a reasonably clear path toward achieving simultaneous compatibility between `ExchangeOnlineManagement` and Azure PowerShell Modules like `Az.Storage`.  There are similar issues with certain `Microsoft.Graph` and `Az.*` modules that I suspect may have a similar root cause.  If anyone else has thoughts on this, please add them to issue 14416.

**Reply by dcaro** [reply]
Thanks, this looks like an assembly conflict. It is too late to address this for the release of next week, but we'll look at it for the next one.
